commands/append.md

```diff
@@ -1,4 +1,6 @@
-If _key_ already exists and is a [Redis string](/docs/data-types/strings), this command appends the _value_ at the
+Appends the _value_ to the [Redis string](/docs/data-types/strings) stored at _key_.
```

Comment:
Appends the _value_ to the [Redis string](/docs/data-types/strings) stored at _key_.

An empty _key_ is equivalent to an empty string (""), so `APPEND` will be similar to `SET` in this case.

Reply:
I don't understand the comment

Reply:
What do you think of replacing lines 1-5 with my version?